.circleci/config.yml

```diff
@@ -110,4 +198,10 @@ workflows:
             ignore: /.*/
           tags:
             only: /v.*/
+    - e2e:
+        requires:
+        - checkout_code
+        filters:
+          tags:
+            only: /v.*/
```

Comment:
Why not run it always?

Reply:
It runs always, but maybe this could be simplified.

It is up to follow how are we going to solve some of build problems.

Reply:
Resolved.